### PR TITLE
Adds Democracy Earth Foundation

### PR DESCRIPTION
**Pull request by medied**

### CODE REVIEW

**Comment by zeke** [comment]
Nice! cc @santisiri @piamancini

🌍

File: README.md [hunk 1]
```diff
@@ -36,3 +36,7 @@ A list of public projects on GitHub tackling immigration, civic engagement, fair
 
 - [IsTrumpStillFuckingPresident.co](https://github.com/zebaslam/TrumpQuotes) - Answers the questions, and provides some joy.
 - [tronalddump.io](https://github.com/tronalddump-io/tronald-app) - Api & web archive for the dumbest things donald trump has ever said.
+
+## Revolution
```
Comment (bkeepers) [inline README.md:40]: Would this fit under the existing heading of "Civic Engagement"?

**Comment by bkeepers** [comment]
Thanks! I'll go ahead an merge as is. I'd love to avoid adding more sections if possible, but I'm not sure this fits anywhere else.

File: README.md (hunk 1 above)
Comment (santisiri) [reply README.md:40]: i think we need to recover also a spirit that dreams of utopias under this context.. revolutions (the good ones, driven by technology and not guns) are possible!

**Pull request merged**